Gemfile

```diff
@@ -29,6 +29,8 @@ gem 'stimulus-rails'
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'jbuilder'
 
+gem 'rails-controller-testing'
+
 # Use Redis adapter to run Action Cable in production
```

Comment:
- [ ] **[OPTIONAL]** Kindly add this gem to your Gemfile for windows users so that your tests can run successfully on any OS.

```ruby
gem 'ffi'
```